api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticSearchService.java

```diff
@@ -53,7 +53,7 @@ public ElasticSearchService(CriteriaDao criteriaDao, Provider<WorkbenchConfig> c
     log.info("Elastic filter: "  + filter.value().toString());
     long count = client().count(new CountRequest(personIndex)
         .source(SearchSourceBuilder.searchSource().query(filter.value())), RequestOptions.DEFAULT).getCount();
-    return new ElasticFilterResponse<>(count, filter.isApproximate());
+    return new ElasticFilterResponse<>(count);
```

Comment:
opt: Probably could remove this response if you want - it might touch a lot more code though, so optional.

Reply:
Done.